### DISCUSSION TITLE
Loyalty NFTs

### DISCUSSION THREAD

**Discussion by peshwar9** [capsule-corp-ternoa/ternoa-proposals#42] [Ideas]
I propose to consider implementing features on Ternoa to support issuance of loyalty NFTs. NFTs can be used to  serve as proof of membership , for exclusive/gated access to specific products/services, improving brand equity, and provide much stronger engagement with  customer base.

What would be the benefit of NFT-based loyalty program for traditional consumer-oriented companies? Cost-effective compared to traditional loyalty programs, better overall experience for consumer, merging digital and physical experiences and even co-create products and brand-experiences with customers using onchain governance.

The product-market fit is strengthened with a Web2 giant recently signing up partnership with a EVM side-chain to provide loyalty features with NFTs, after what I assume would have been thorough due diligence.

What do you think? What kind of loyalty features can NFTs provide?

**Comment by Crypte** [comment]
I think it's a great idea, and would be of real interest when you see the complexity of current loyalty systems in many companies. As you mentioned, it could provide a (nearly) turnkey solution for smaller companies that don't necessarily have the means to develop it or pay a company to issue and manage it. 

However, using this system requires a customer base capable of interacting with a wallet/NFT, which is not yet a reality. That's why I had in mind to propose the implementation of the simplified wallet creation system that we have in the beta of the time capsule dapp (private key in the form of a QRcode encrypted by the answer to a personal question) directly in the Ternoa app wallet. This would provide a more intuitive and therefore more accessible experience.

**Reply by Herrlalwani** [reply]
@Gitcrypte this is a great idea, maybe you could open a new discussion thread around this?
After all we all are looking to simplify the user experience.

**Comment by Herrlalwani** [comment]
I see a world in which the Loyalty NFTs are a passport, free to acquire and initially empty, and it gets stamped as the customers/users engage with the brand in order to unlock rewards.
Two usecases: 

- Liquidity for the customers/users, who can monetize their loyalty/effort by selling the stamps or the passport containing them.
- Interoperability, as one brand can leverage a customer's loyalty and interactions with another brand, this will open up a world of infinite possibilities

Another important aspect is the ability for brands to develop these programs/dApps/projects without much of technical know-how.

Here is Quant briefly talking about loyalty program and their new token standard QRC-721: https://quant.network/perspectives/unlocking-the-full-potential-of-nfts-with-qrc-721/
